### DISCUSSION TITLE
websocket支持

### DISCUSSION THREAD

**Discussion by Leibnizhu** [vertx-china/tree-new-bee#11] [Ideas]
需要稍微重构下server

**Comment by chengenzhao** [comment]
可以先行实现web socket
然后测试一下web socket的连接
进而考虑如何抽取重复代码
但是目前web socket的进展缓慢，没有看到有客户端或者网页端支持
可以在群里问一下，谁打算做web socket的客户端或者网页端

**Comment by Leibnizhu** [comment]
已支持
